Source/WebKit/NetworkProcess/webtransport/cocoa/NetworkTransportStreamCocoa.mm

```diff
@@ -24,25 +24,40 @@
  */
 
 #import "config.h"
-#import "NetworkTransportReceiveStream.h"
+#import "NetworkTransportStream.h"
 
 #import "NetworkTransportSession.h"
 #import <wtf/BlockPtr.h>
 #import <wtf/cocoa/SpanCocoa.h>
+#import <wtf/cocoa/VectorCocoa.h>
 
 namespace WebKit {
 
-NetworkTransportReceiveStream::NetworkTransportReceiveStream(NetworkTransportSession& session, nw_connection_t connection)
+NetworkTransportStream::NetworkTransportStream(NetworkTransportSession& session, nw_connection_t connection, NetworkTransportStreamType streamType)
     : m_identifier(WebTransportStreamIdentifier::generate())
     , m_session(session)
     , m_connection(connection)
+    , m_streamType(streamType)
 {
     ASSERT(m_connection);
-    receiveLoop();
+    ASSERT(m_session);
+    if (m_streamType != NetworkTransportStreamType::OutgoingUnidirectional)
+        receiveLoop();
 }
 
-void NetworkTransportReceiveStream::receiveLoop()
+void NetworkTransportStream::sendBytes(std::span<const uint8_t> data, bool)
 {
+    if (m_streamType == NetworkTransportStreamType::IncomingUnidirectional)
```

Comment:
Let's change this to ASSERT(m_streamType != NetworkTransportStreamType::IncomingUnidirectional) like NetworkTransportStream::receiveLoop